machine_spec.py

```diff
@@ -284,7 +286,8 @@ def detect_arch() -> str:
     "armbe8",
     "mips",
     "mips64",
+    "powerpc",
```

Comment:
```suggestion
    "ppc",
    "ppc64",
```

(Suppose we might as well add `ppc64` while at it.)